msbuild/Xamarin.MacDev.Tasks/Tasks/FilterStaticFrameworksTaskBase.cs

```diff
@@ -32,6 +34,11 @@ public override bool Execute ()
 							continue;
 						}
 
+						if (OnlyFilterFrameworks && !Path.GetDirectoryName (frameworkExecutablePath).EndsWith (".framework", StringComparison.OrdinalIgnoreCase)) {
+							Log.LogMessage (MessageImportance.Low, $"Skipped processing {item.ItemSpec} because it's not a framework");
```

Comment:
I wonder if this should be logged as a warning instead. I know it could be annoying, but if we are choosing to ignore/skip static libraries, it means those are not supported in the current context. 
And a warning could be a hint for crashes that may happen because those were not included in the app bundle?